src/service/resources-impl.js

```diff
@@ -742,7 +742,7 @@ export class Resources {
         if (diff == 0) {
```

Comment:
It could be argued either way, we just need to be consistent with how we resolve for all the noop cases.